code/modules/vore/resizing/resize_vr.dm

```diff
@@ -243,10 +243,18 @@ var/const/RESIZE_A_SMALLTINY = (RESIZE_SMALL + RESIZE_TINY) / 2
 	if(a_intent == I_HELP) // Theoretically not possible, but just in case.
 		return FALSE
 
+	if(tmob.a_intent != I_HELP && prob(35))
+		to_chat(pred, "<span class='danger'>[prey] dodges out from under your foot!</span>")
+		to_chat(prey, "<span class='danger'>You narrowly avoid [pred]'s foot!</span>")
+		return FALSE
+
 	now_pushing = 0
 	forceMove(tmob.loc)
 	if(a_intent == I_GRAB || a_intent == I_DISARM)
-		tmob.resting = 1
+		if(tmob.a_intent = I_HELP)
```

Comment:
This is giving an "Assignment in condition" warning, and I think == would make more sense here.